### DISCUSSION TITLE
eqMac v1.2.0

### DISCUSSION THREAD

**Discussion by nodeful** [bitgapp/eqMac#544] [General]
# v1.2.0 - Improved Stability & Performance + New Features
### Features:
* Entire App Bypass switch
* UI Scaling (0.5x - 2x scaling)
* Spatial Audio (Pro)
* Expert EQ Controls revamp - Mixer style
* Expert EQ Style - Graph, Sliders or Both
* Beta program update feed

### Stability & Performance:
* Fixed bug where Window would open on Startup
* Improvements to volume handling
* Improvements device handling
* Improvements to Sleep/Wake cycle handling
* All UI Controls performance improvements
* Expert EQ Spectrum Analyzer performance, resource usage, utility improvements
* EQ Type switching fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/bitgapp/eqMac/releases/tag/v1.2.0'>eqMac v1.2.0</a>.</em>

**Comment by Geffreyvanderbos** [comment]
I've been using the release for about 2 hours now.

It's great. The UI is snappy. The CPU usage is notably less. I didn't have any need restarting it (also when switching outputs and recalling the Macbook Air M1 2020 from sleep).

I occasionally mouse scroll over the UI and unintentionally change things. (Might be nice to have a 'click to active' kind of pattern, before scrolling.)

You make my WH-1000XM3 sound at least 5x better.

Thank you for your hard work. Is there any way I can do a one time donation?

**Comment by PapiPeppers** [comment]
Incredible update! Noticeably better sound on all fronts. Far less buggy than initial release. Great work!
